### DISCUSSION TITLE
Built-in alert manager should be visible even when it is the only one alertmanager datasource

### DISCUSSION THREAD

**Discussion by devminded** [grafana/grafana#45464] [Alerting]
### Issue
When there is only one alert manager present (the embedded one) it is not visible in the alerting panels.
The drop down should be visible even when there is only one present (but could be greyed out/uneditable).

This makes things clearer, especially since the documentation for Unified Alerting starts by mentioning that users should select an alert manager in the drop down.

**Single alert manager**
![contact-only-one-alertmanager](https://user-images.githubusercontent.com/10235454/154271187-8aeed38e-da4f-4a02-9c3c-45932968feef.png)

**With several alert manager**
![contact-with-alertmanager](https://user-images.githubusercontent.com/10235454/154271223-09e25709-1d37-47ac-b81e-afed4e818f71.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
